### PR TITLE
docs(readme): wordsmithing

### PR DESCRIPTION
**Pull request by alexcg1**

### CODE REVIEW

**Review by theUnkownName — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ docker run -v "$(pwd)/j:/j" jinaai/jina hello-world --workdir /j && open j/hello
 </p>
 
 </details>
-It downloads the Fashion-MNIST training and test dataset and tells Jina to index 60,000 images from the training set. Then it randomly samples images from the test set as queries and asks Jina to retrieve relevant results. The whole process takes about 1 minute, and eventually opens a webpage and shows results like this:
+This downloads the Fashion-MNIST training and test dataset and tells Jina to index 60,000 images from the training set. Then it randomly samples images from the test set as queries and asks Jina to retrieve relevant results. The whole process takes about 1 minute, and after running opens a webpage and shows results:
```
Comment (cristianmtr) [inline README.md:88]: `..., after which it opens a webpage ...`, maybe

**Review by hanxiao — DISMISSED** [review]

**Comment by alexcg1** [comment]
@hanxiao let's hold off on the merge. I'm going through the code and testing to ensure we're not missing anything

**Comment by alexcg1** [comment]
All changes finished now

**Pull request merged**